crmsh/ui_resource.py

```diff
@@ -704,6 +704,19 @@ def _trace_op_interval(self, context, rsc_id, rsc, op, interval, dir):
     @command.completers(compl.primitives, _raoperations)
     def do_trace(self, context, rsc_id, *args):
         'usage: trace <rsc> [<op>] [<interval>] [<log-dir>]'
+        def get_ha_varlib_value():
+            """
+            Find the value of HA_VARLIB, which should be defined in ocf-directories
+            """
+            _, out, _ = ShellUtils().get_stdout_stderr("rpm -ql resource-agents")
```

Comment:
This will not work when resource-agents are installed from source. We should not depends on package manager here.

Reply:
I think we should enumerate possible libdirs, just like setting up `PATH` in `main.envsetup`.